CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ set(BRION_MAINTAINER_EMAIL "<bbp-open-source@googlegroups.com>")
 set(BRION_MAINTAINER "${BRION_MAINTAINER_NAME} ${BRION_MAINTAINER_EMAIL}")
 set(BRION_LICENSE LGPL)
 set(BRION_DEB_DEPENDS libhdf5-serial-dev libboost-date-time-dev
-  libboost-filesystem-dev libboost-regex-dev libboost-system-dev
+  libboost-filesystem-dev  libboost-iostreams-devlibboost-regex-dev libboost-system-dev
```

Comment:
There's still a missing space and now that I remember, please update also the list on CMake/CPackConfig.cmake (This one is the list of dependencies for building, the other list is the actual dependencies for the deb package to generate)